qir/qat/ValidationPass/ValidationPassConfiguration.cpp

```diff
@@ -61,7 +61,46 @@ namespace quantum
             profile.allowed_pointer_types_   = {"i8*",    "i16*",    "i32*",    "i64*",
                                               "Qubit*", "Qubit**", "Result*", "Result**"};
         }
-        else if (name == "quantinuum" || name == "rigetti" || name == "qci")
+        else if (name == "provider_4bf9")
+        {
+            profile.allow_internal_calls_     = false;
+            profile.allowlist_external_calls_ = true;
+            profile.allowlist_opcodes_        = true;
+            profile.opcodes_        = Set{"ret", "br", "phi", "add", "sub", "mul", "ashr", "and", "or", "xor"};
```

Comment:
I believe this profile should include `fadd`, `fsub`, `fmul`, `icmp` and `fcmp` as well.